packages/openneuro-components/src/dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import './dropdown.scss'
+
+export interface DropdownProps {
+  label: Record<string, any>
+  children
+}
+
+export const Dropdown = ({ children, label }: DropdownProps) => {
+  const [isOpen, setOpen] = React.useState(false)
+  return (
+    <div className="dropdown-wrapper" onClick={() => setOpen(!isOpen)}>
```

Comment:
Since the new value of `isOpen` is being set based on what its previous value is, it's better practice to pass in a callback that uses the previous `isOpen` value instead of using `isOpen` directly.

ie `onClick={() => setOpen(prevIsOpen => !prevIsOpen)}`

Also the setter function should be called `setIsOpen`